test/acceptance/testing.go

```diff
@@ -147,18 +158,33 @@ func Run(t *testing.T, c AccTestCase) {
 		t.Fatal(err)
 	}
 
-	err := os.Chdir(c.Path)
-	if err != nil {
-		t.Fatal(err)
-	}
 	if c.OnStart != nil {
 		c.OnStart()
 	}
-	err = terraformApply()
+
+	// Disable terraform version checks
+	// @link https://www.terraform.io/docs/commands/index.html#upgrade-and-security-bulletin-checks
+	checkpoint := os.Getenv("CHECKPOINT_DISABLE")
+	os.Setenv("CHECKPOINT_DISABLE", "true")
+
+	// Execute terraform init if .terraform folder is not found in test folder
+	err := c.terraformInit()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = c.terraformApply()
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer terraformDestroy()
+
+	defer func() {
+		os.Setenv("CHECKPOINT_DISABLE", checkpoint)
+		err := c.terraformDestroy()
```

Comment:
Terraform doesn't do any checks in a `terraform destroy` ?
Should we instead reset the `CHECKPOINT_DISABLE` after the destroy ?

Reply:
> Terraform doesn't do any checks in a terraform destroy ?

I don't know

> Should we instead reset the CHECKPOINT_DISABLE after the destroy ?

Sure this is a nice catch thanks !

Reply:
Fixed